### DISCUSSION TITLE
Allow local http to nextcloud

### DISCUSSION THREAD

**Discussion by storm1er** [nextcloud/all-in-one#1798] [Ideas] [answered]
### Is your feature request related to a problem? Please describe. 

- Nextcloud is installed locally on a RPI
- Worldwide access is done using cloudflare tunnel (working perfectly)
- But I can't access locally to my nextcloud (means that event on the same network, I'm going through cloudflare)
- This cause a massive drop on network performance due to the routing being worldwide instead of local (+ no internet meaning we can't access nextcloud :/ )

### Describe the solution you'd like

I would like to access my nextcloud instance locally, either by:
- using a static route on my router/computer (registering static route `domain.tld` to my RPi IP address)
- using http://rpi-ip:port/ (or https://rpi-ip:port/)

### Describe alternatives you've considered

I tried to set APACHE_PORT to 443/80 and added a static route, but doesn't work due to the missing certificate (obviously)

### Additional context

I thought about adding another reverse proxy locally. Still, as I can't register the domain to 2 IP addresses (as I'm using Cloudflare for worldwide connection), I'll be unable to use quick cert' providers like, let's encrypt or else.

**Comment by szaimen** [comment, marked as answer]
Hi, recommended is to use the same domain to connect to the server by following https://github.com/nextcloud/all-in-one#how-can-i-access-nextcloud-locally
